atm.rb

```diff
@@ -1,13 +1,64 @@
+# ATM by Eric Anderson for evaluation
 class Atm
-  STARTING_BALANCE = 100.0
-
-  def initialize(filename = "balance.txt")
+  def initialize(filename = 'balance.txt')
     @filename = filename
   end
 
   def run
-    balance = STARTING_BALANCE
+    balance = 100
+    if File.exist?(@filename)
+      @account = File.open("#{@filename}", 'r+')
+      balance = @account.to_a.last.to_f
```

Comment:
Your version is nice, as it safes the history of the balances.

It is different than specified: `This file contains a single line with the customer’s starting balance.` but actually nicer :wink: